Core/GameEngine/Source/Common/System/LocalFile.cpp

```diff
@@ -412,6 +444,32 @@ Int LocalFile::writeFormat( const WideChar* format, ... )
 	return write( buffer, length * sizeof(WideChar) );
 }
 
+//=================================================================
+// LocalFile::writeChar - Ascii
+//=================================================================
+
+Int LocalFile::writeChar( const Char* character )
```

Comment:
I think this should return the result of `write` instead of the character that was passed, or EOF, which is not an intuitive result.

Reply:
Ok makes sense then.